de.metas.adempiere.adempiere/base/src/main/java/org/adempiere/impexp/BPartnerImportProcess.java

```diff
@@ -133,7 +131,6 @@ protected ImportRecordResult importRecord(final IMutable<Object> state, final I_
 		{
 			// create a new list because we are passing to a new partner
 			context.sameBPpreviousImportRecords = new ArrayList<>();
```

Comment:
don't access private fields directly, even if you can.
better do something like:
```
context.clearPreviousRecords();
```